src/Utils/Helpers.ts

```diff
@@ -46,6 +46,20 @@ export class Helpers {
     };
   }
 
+  public static getDelimitedInput(
```

Comment:
Let's move this method to `Readline`. I'm not sure we will use it somewhere else